command/agent.go

```diff
@@ -206,6 +207,29 @@ func (c *AgentCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Check if the `-address` flag was set on the command
+	var vaultAddrFlagSet bool
+	f.Visit(func(f *flag.Flag) {
+		if f.Name == "address" {
```

Comment:
Since this is used in a coupe places can you make this a const a la the diff I shared with you yesterday?

Reply:
Fixed.